src/Octoliner.h

```diff
@@ -14,7 +14,8 @@
 #ifndef __OCTOLINER_H__
 #define __OCTOLINER_H__
 
-static constexpr uint8_t INPUT_PULLDOWN = 0x3;
+#define INPUT_PULLDOWN 0x3
```

Comment:
```suggestion
#define INPUT_PULLDOWN 0x3
```

Why INPUT_PULLDOWN changed be `define`?
May be use `constexpr.`

Reply:
This will create confusion in the code. There is an iteration of the constants INPUT, OUTPUT, INPUT_PULLUP and then suddenly inputPullDown. And the rest of the constants are standard Arduino constants.

Reply:
Ok)